source/SpinalHDL/Other language features/formal.rst

```diff
@@ -0,0 +1,89 @@
+
+Formal
+==========
+
+General
+-------
+
+There is limited support for SystemVerilog Assertions.
+
+To add Assertions to a Component for formal verification, repeating the practical example from before:
```

Comment:
This sentence doesn't feels right... It feels like there is something missing (`repeating the practical example from before` reference an example that is not there)
maybe:
```suggestion
You can add formal statement (assume, assert ecc..) in the Component definition, like the example below:
```

Reply:
I meant the example from the assertion, but your sentence is much clearer.